### PR TITLE
Consistent autocomplete layout

### PR DESCRIPTION
**Pull request by ecotopian**
https://ecosia.atlassian.net/browse/MOB-1517

### CODE REVIEW

**Comment by ecotopian** [comment]
I got feedback from Yann and Layout is now implemented to look like this:
Simulator Screen Shot - iPhone 13 - 2022-06-22 at 15 48 04

File: Client/Frontend/Browser/SearchViewController.swift [hunk 1]
```diff
@@ -628,26 +639,33 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
...
                 twoLineCell.backgroundColor = UIColor.theme.ecosia.autocompleteBackground
                 twoLineCell.accessoryView = nil
+                let imageName = isBookmark ? "bookmarksEmpty" : "libraryHistory"
+                twoLineCell.leftImageView.contentMode = .center
+                twoLineCell.leftImageView.image = .init(named: imageName)
+                twoLineCell.leftImageView.tintColor = .theme.ecosia.secondaryText
                 cell = twoLineCell
             }
         }
+        //(cell as? Themeable)?.applyTheme()
```
Comment (jman82) [inline Client/Frontend/Browser/SearchViewController.swift:668]: Does this change also require an "Ecosia" tag?

**Review by jman82 — APPROVED** [review]
Looks great! :rocket:

**Review by vauxhall — COMMENTED** [review]
👍

**Pull request merged**